README.markdown

```diff
@@ -241,13 +315,37 @@ func setAge(age: Int) {
 }
 ```
 
-## Closures
+## Closure Expressions
+
+Use trailing closure syntax only if there's a single closure expression parameter at the end of the argument list. Give the closure parameters descriptive names.
 
-Use trailing closure syntax wherever possible. In all cases, give the closure parameters descriptive names:
+**Preferred:**
+```swift
+UIView.animateWithDuration(1.0) {
+  self.myView.alpha = 0
```

Comment:
Please remove **self** from here. Self references are done only when needed as described in the **Use of Self** section.